ksql-udf-quickstart/src/main/resources/archetype-resources/pom.xml

```diff
@@ -72,6 +73,11 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- JUnit 5 requires Surefire version 2.22.1 or higher -->
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.1</version>
```

Comment:
nit: let's put the version in the properties section

Reply:
done :)